src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -37,7 +37,13 @@ public abstract partial class DelegatingHandler : System.Net.Http.HttpMessageHan
     }
     public partial class FormUrlEncodedContent : System.Net.Http.ByteArrayContent
     {
-        public FormUrlEncodedContent(System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string?, string?>> nameValueCollection) : base (default(byte[])) { }
+        public FormUrlEncodedContent(
+            [System.Diagnostics.CodeAnalysis.DisallowNullAttribute]
```

Comment:
I know I suggested this,  but I think it should work fine without the attribute,  too.

Reply:
It does, will remove it